pub_key.go

```diff
@@ -38,11 +43,9 @@ var _ PubKey = PubKeyEd25519{}
 // Implements PubKeyInner
 type PubKeyEd25519 [32]byte
 
+// Address is the SHA256-20 of the raw pubkey bytes.
 func (pubKey PubKeyEd25519) Address() Address {
-	// append type byte
-	hasher := ripemd160.New()
-	hasher.Write(pubKey.Bytes()) // does not error
-	return Address(hasher.Sum(nil))
+	return Address(tmhash.Sum(pubKey[:]))
```

Comment:
Just to make sure, as we dropped ed25519 support for wallets, we still need the `Address()` of a `PubKeyEd25519` for other purposes, right?

Reply:
Huh? Just saw this tidbit here...
Is that true? Using only secp? It would be good to add that to BreakingChanges...

Reply:
will still be supported in go-crypto, we're just not going to have tooling for it for user keys at launch